Assignment2.txt

```diff
@@ -0,0 +1,13 @@
+Name: Anya Lauria 
+
+1. If the operators are +-/*, then yes, they can be. 
+2. Syntax refers to the grammar of a programming language. Therefore, a 
+   syntax  error refers to failure in conforming to the grammar of a
+   particular language, which would result in a compiler being unable to
+   compile the program. On the other hand, a logic error is when a 
+   program does not understand what it is expected to do. 
+3. True
```

Comment:
You're right, but remember that there are no boolean values in C! So the answer is technically 1.